src/load-modules.ts

```diff
@@ -53,6 +59,15 @@ const nameFormatters: Record<string, NameFormatter> = {
   camelCase: (s) => camelCase(s),
 }
 
+export interface LoadModulesResult {
```

Comment:
Sorry to be pedantic about this but could you add comments to the interfaces and functions?

Reply:
Of course! Good point!